target/linux/ath79/dts/qca9563_tplink_re450-v3.dts

```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
```

Comment:
Please create a shared DTSI, e.g. qca9563_tplink_re450.dtsi for v2 and v3. As it looks, you would only contain the compatible, model and spi in the DTS files then ...

Reply:
Done.

Reply:
Done.